package.json

```diff
@@ -7,7 +7,7 @@
   "license": "MIT",
   "scripts": {
     "start": "nodemon src/app.js",
-    "test": "export IS_JEST=1 && yarn validate && jest --passWithNoTests",
+    "test": "set IS_JEST=1 && yarn validate && jest --passWithNoTests",
```

Comment:
@mkkasem if you look at the discussion between me and Shrreya on the #23 PR you will get a better view on this. 
`set` doesn't work in github, you need to use it locally but when you commit make sure that it is `export` not `set` 
Just when you are going to push them to github after committing you can revert it back to `set` so it works on you terminal